content/authors/marcel_rd.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Marcel Ribeiro-Dantas
+avatar: marcel.jpg
+---
+
+Scientist at [Institut Curie](https://intstitut-curie.org).
```

Comment:
Can we add a bit more details? we'll make names clickable soon, so Twitter, other accounts, bio, can make the visibility for authors better.

Reply:
@mribeirodantas what do you want in your bio?

Reply:
Hm.. Good question. My Twitter and website would be good.

For description, instead of "Scientist at", I think it's better to say "Early Stage Researcher at". 

Do you want a small bio?

Early Stage Researcher at Institut Curie with over 10 years of experience in the field of biomedical engineering and health informatics. Areas of interest include Causal Inference, Artificial Intelligence, and Data Science. Degrees in Computer and Automation Engineering (Eng), Big Data (Grad degree), and Bioinformatics (MSc). Currently enrolled in a Ph.D. at EDITE (Sorbonne Université).

What do you think?

Reply:
yeah that's good!